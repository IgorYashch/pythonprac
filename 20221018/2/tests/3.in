:pow_func x y pow(x,y)
:perm_func a b perm(a,b)
pow_func 2 5
perm_func 10 3
quit   "{}:{}"