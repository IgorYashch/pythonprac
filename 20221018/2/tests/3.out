32.0
720
3:5
